integration_tests/pyproject.toml

```diff
@@ -20,6 +20,7 @@ python-dateutil = "^2.8.1"
 pystarport = { git = "https://github.com/crypto-com/pystarport.git", branch = "main" }
 web3 = "^5.20.1"
 eth-bloom = "^1.0.4"
+pytest-xdist = "1.34.0"
```

Comment:
we didn't use pytest-xdist to run the integration tests, so don't need the dependency.

Reply:
Why we don't use it? Too clumsy?

Reply:
https://github.com/crypto-org-chain/cronos/pull/201#discussion_r741622869